nginx/public/node/frontend/public/js/app/search/chant-search/about-volpiano.template.html

```diff
@@ -21,47 +21,79 @@
+                g---h</span></div></div>
+            </div>
+        <div class="row">
+        <p>No separation occurs between pitches in the same neume. One hyphen ("-") separates
+            neumes. Two hyphens ("--") separate neumes associated with different syllables.
+            Three hyphens ("---") separate neumes associated with different words.
+        </p>
+    </div>
     <div class="row">
         <p><b>Features of Volpiano Search</b></p>
         <ol>
-            <li>
+            <li style="margin-bottom:20pt">
                 There are two types of Volpiano search in Cantus Ultimus:
-                "Volpiano" and "Volpiano (Literal)." "Volpiano (Literal)"
-                matches the way pitches in the query are divided into neumes. In
-                Volpiano, a hyphen ("-") separates pitches belonging to
-                different neumes. For example, the two queries below will yield
-                the same results in a "Volpiano" search, but not in a "Volpiano
-                (Literal)" search. In a "Volpiano (Literal)" search, the first
-                query will only return chants in which those four pitches occur
-                a single neume, and the second will only return chants where the
-                pitches occur in two consecutive two-pitch neumes.
+                "Volpiano" and "Volpiano (Literal)." "Volpiano" matches a 
+                sequence of pitches, regardless of how those pitches are grouped 
+                into neumes, syllables, and words. "Volpiano (Literal)" matches 
+                the pitches in the query and the way those pitches are grouped. 
+                For example, the two queries below will return the same results 
+                in a "Volpiano" search, but different subsets of those 
+                results in a "Volpiano (Literal)" search. In a "Volpiano (Literal)" 
+                search, the first query will only return chants 
+                in which the sequence of four pitches shown occurs in a single neume, 
+                while the second will only return chants where the pitches occur 
+                in two separate consecutive neumes. 
                 <div class="volpiano" style="text-align:center"><div
                         class="volpiano-syllable">1---fgfh---33</div><div
                         class="volpiano-syllable">
                         1---fg-fh---
                     </div>
                 </div>
+                Note that in these queries, chants may be returned where the pitches 
+                do not make up a complete neume. For example, a "Volpiano (Literal)" 
+                search of the second query above could return a chant that includes:
+                <div class="volpiano" style="text-align:center"><div
+                        class="volpiano-syllable">1---efg-fhgf---</div>
+                </div>
+                <!---
```

Comment:
Yep, I put it here because I'll uncomment (or not) in the future depending on the results of some discussion on the issue page (where I've posted the details), and since I already wrote it I thought I'd just keep it in for now.